### PR TITLE
Fixed "All categories selecting when editing wizard" + Refactored some stuff

### PR DESCRIPTION
**Pull request by iartemiev**
## What's this PR do?
* Fixes the bug in the title of this PR
* Refactored some of `AddContact.js` component

## Where should the reviewer start?
`AddContact.js`, `cboWizardActions.js`, `cboWizardReducer.js`

## How should this be manually tested?
1. Employers -> TJD
2. +New Organization
3. Fill out required fields
4. Select a category or two
5. Click the **Next** button
6. Entered Req'd data & click **Next**
7. Click **Edit Contact Details**
8. Scroll down to the categories
9. Only the ones originally selected should be checked.

### CODE REVIEW

**Comment by aljordan82** [comment]
I am getting an error when updating contacts
```
Uncaught TypeError: this.buildContactObject is not a function
    at AddContact.saveAdditionalContact (eval at <anonymous> (cbo.js:19151), <anonymous>:249:29)
    at saveFunction (eval at <anonymous> (cbo.js:19151), <anonymous>:349:25)
    at Object.ReactErrorUtils.invokeGuardedCallback (eval at <anonymous> (cbo.js:3622), <anonymous>:69:16)
    at executeDispatch (eval at <anonymous> (cbo.js:3574), <anonymous>:85:21)
    at Object.executeDispatchesInOrder (eval at <anonymous> (cbo.js:3574), <anonymous>:108:5)
    at executeDispatchesAndRelease (eval at <anonymous> (cbo.js:1936), <anonymous>:43:22)
    at executeDispatchesAndReleaseTopLevel (eval at <anonymous> (cbo.js:1936), <anonymous>:54:10)
    at Array.forEach (<anonymous>)
    at forEachAccumulated (eval at <anonymous> (cbo.js:5904), <anonymous>:24:9)
    at Object.processEventQueue (eval at <anonymous> (cbo.js:1936), <anonymous>:257:7)
```

**Comment by iartemiev** [comment]
Ah, I see what's happening. I'll fix it shortly

**Comment by aljordan82** [comment]
Another issue, Validation messages are not showing under form fields.

File: gulp/src/cbo/components/cbo/AddContact.js [hunk 1]
```diff
@@ -358,6 +342,7 @@ class AddContact extends Component {
               label="Address Line 2">
               <TextInput
                 name="address2"
+                onChange={(event) => this.handleChange(event)}
```
Comment (aljordan82) [inline gulp/src/cbo/components/cbo/AddContact.js]: Can you define the bound handleChange in the constructor?

**Pull request merged**